internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -0,0 +1,269 @@
+			definitionPaths = append(definitionPaths, filepath.Join(definitionsDir, fileInfo.Name()))
+		}
+	}
+	return definitionPaths, nil
+}
+
+func verifyKindContext() error {
+	logger.Debug("ensure that kind context is selected")
+
+	cmd := exec.Command("kubectl", "config", "current-context")
+	errOutput := new(bytes.Buffer)
+	cmd.Stderr = errOutput
+	output, err := cmd.Output()
+	if err != nil {
+		return errors.Wrapf(err, "kubectl command failed (stderr=%q)", errOutput.String())
+	}
+	currentContext := string(bytes.TrimSpace(output))
+
+	if currentContext != "kind-kind" {
+		return fmt.Errorf("unexpected kubectl context selected (actual: %s, expected: %s)", currentContext, kindContext)
+	}
+	return nil
+}
+
+func findKindControlPlane() (string, error) {
+	logger.Debugf("find \"%s\" container", kindControlPlaneContainerName)
+
+	cmd := exec.Command("docker", "ps", "--filter", "name=kind-control-plane", "--format", "{{.ID}}")
+	errOutput := new(bytes.Buffer)
+	cmd.Stderr = errOutput
+	output, err := cmd.Output()
+	if err != nil {
+		return "", errors.Wrapf(err, "could not find \"%s\" container to the stack network (stderr=%q)", kindControlPlaneContainerName, errOutput.String())
+	}
+	containerIDs := bytes.Split(bytes.TrimSpace(output), []byte{'\n'})
+	if len(containerIDs) != 1 {
+		return "", fmt.Errorf("expected single %s container, make sure you have run \"kind create cluster\" and the %s container is present", kindControlPlaneContainerName, kindControlPlaneContainerName)
+	}
+	return string(containerIDs[0]), nil
+}
+
+func connectControlPlaneToElasticStackNetwork(controlPlaneContainerID string) error {
+	stackNetwork := fmt.Sprintf("%s_default", stack.DockerComposeProjectName)
+
+	logger.Debugf("check network connectivity between service container %s (ID: %s) and the stack network %s", kindControlPlaneContainerName, controlPlaneContainerID, stackNetwork)
+	cmd := exec.Command("docker", "network", "inspect", stackNetwork)
+	errOutput := new(bytes.Buffer)
+	cmd.Stderr = errOutput
+	output, err := cmd.Output()
+	if err != nil {
+		return errors.Wrapf(err, "could not inspect the stack network (stderr=%q)", errOutput.String())
+	}
+
+	var networkDescriptions []struct {
+		Containers map[string]struct {
+			Name string
+		}
+	}
+	err = json.Unmarshal(output, &networkDescriptions)
+	if err != nil {
+		return errors.Wrapf(err, "can't unmarshal network inspect for %s (stderr=%q)", stackNetwork, errOutput.String())
+	}
+
+	if len(networkDescriptions) != 1 {
+		return fmt.Errorf("expect single network inspect record, got %d entries", len(networkDescriptions))
+	}
+
+	for _, c := range networkDescriptions[0].Containers {
+		if c.Name == kindControlPlaneContainerName {
+			logger.Debugf("container %s is already attached to the %s network", kindControlPlaneContainerName, stackNetwork)
+			return nil
+		}
+	}
+
+	logger.Debugf("attach service container %s (ID: %s) to stack network %s", kindControlPlaneContainerName, controlPlaneContainerID, stackNetwork)
```

Comment:
```suggestion
	logger.Debugf("attach kind control plane container %s (ID: %s) to stack network %s", kindControlPlaneContainerName, controlPlaneContainerID, stackNetwork)
```

Reply:
I adjusted the comment to the similar form.